CONTRIBUTING.md

```diff
@@ -48,6 +48,12 @@ To clone and build iTwinUI, you'll need [Git](https://git-scm.com), [Node 18+](h
 
 _Before running this command, make sure Docker is running. See [Visual testing](#visual-testing-css) (CSS and React) sections below for more details._
 
+### To approve test image changes
+
+* `pnpm approve` (approve in all workspaces)
+* `pnpm approve:css` (approve in css-workshop only) ([more info](#how-to-run-tests))
+* `pnpm approve:react` (approve in react-workshop only) ([more info](#running-visual-tests))
+
```

Comment:
I think this clutters to the intro section, the purpose of which is to list out the most important commands (for quick glance).

It might be better to just show `pnpm approve` here, and document the full approving workflow with more details further down the page.

Reply:
The intro can be made even more succint (notice the addition of "all" in the heading):

> ### To approve all test image changes
>
> `pnpm approve` ([more info](#visual-testing-approval))


And the later section can be made more descriptive.

> ### Visual testing (approval)
>
> After making any desired visual changes, the following commands can be used to update the test image snapshots:
>
> * `pnpm approve` (approve in all workspaces)
> * `pnpm approve:css` (approve in css-workshop only) ([more info](#how-to-run-tests))
> * `pnpm approve:react` (approve in react-workshop only) ([more info](#running-visual-tests))